planner/core/casetest/integration_test.go

```diff
@@ -3205,8 +3205,8 @@ func TestIssue32632(t *testing.T) {
 		"`S_ACCTBAL` decimal(15,2) NOT NULL," +
 		"`S_COMMENT` varchar(101) NOT NULL," +
 		"PRIMARY KEY (`S_SUPPKEY`) /*T![clustered_index] CLUSTERED */)")
-	tk.MustExec("analyze table partsupp;")
-	tk.MustExec("analyze table supplier;")
```

Comment:
If I don't modify the test, I would get the error as follows:
```
Diff:
--- Expected
+++ Actual
@@ -4,4 +4,4 @@
 [    └─HashAgg 1.00 mpp[tiflash]  funcs:sum(test.partsupp.ps_supplycost)->Column#15]
-[      └─Projection 12500.00 mpp[tiflash]  test.partsupp.ps_supplycost]
-[        └─HashJoin 12500.00 mpp[tiflash]  inner join, equal:[eq(test.supplier.s_suppkey, test.partsupp.ps_suppkey)]]
+[      └─Projection 8000000000.00 mpp[tiflash]  test.partsupp.ps_supplycost]
+[        └─HashJoin 8000000000.00 mpp[tiflash]  inner join, equal:[eq(test.supplier.s_suppkey, test.partsupp.ps_suppkey)]]
 [          ├─ExchangeReceiver(Build) 10000.00 mpp[tiflash]  ]
```
Before the PR, in `getColumnNDV`, since `hist.Count` is 0, we use `float64(ds.statisticTable.RealtimeCount) * distinctFactor` to calculate NDV. After the PR, if we still analyze `partsupp` and `supplier` when they don't have any data, since `hist.IsStatsInitialized()` is true, we use `float64(hist.Histogram.NDV)` to calculate NDV. In another word, before the PR we use `hist.Count > 0` to decide whether the table is analyzed while after the PR we use `float64(hist.Histogram.NDV)` to decide that. I think the latter is more reasonable. I remove the two analyze commands to keep cardinality estimation the same as before. Note that it is rare in the reality that `RealtimeCount` reaches 800000/10000 but auto analyze is not triggered.